src/main/java/mate/academy/bookstore/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.bookstore.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import mate.academy.bookstore.validation.FieldMatch;
+
+@Data
+@FieldMatch
+public class UserRegistrationRequestDto {
+    @Email
+    private String email;
+
+    @NotNull
+    private String password;
+
+    @NotNull
+    private String repeatPassword;
+
+    @NotNull
```

Comment:
What if I pass empty string here? Also, what if I pass string which is longer then allowed length in DB?